Authentication/RecoverPassword/RecoverPasswordController.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  RecoverPasswordController.swift
+//  Authentication
+//
+//  Created by Daniela Riesgo on 3/18/16.
+//  Copyright © 2016 Wolox. All rights reserved.
+//
+
+import Foundation
+
+
+public final class RecoverPasswordController: UIViewController {
```

Comment:
This shouldn't be part of this PR it is out of the scope of the card. You can leave it but next time only do what is necessary for this feature to work.